common/src/main/java/tachyon/MasterClientBase.java

```diff
@@ -44,6 +48,8 @@
   public MasterClientBase(InetSocketAddress masterAddress, TachyonConf tachyonConf) {
     super(masterAddress, tachyonConf, "master");
     mUseZookeeper = mTachyonConf.getBoolean(Constants.ZOOKEEPER_ENABLED);
+    mCompatibleVersions = new HashSet<Long>();
```

Comment:
I moved these to the tachyon.Version class. It will be maintain (manually) there.